rust/element/src/cursor.rs

```diff
@@ -171,6 +171,20 @@ impl<'a> Cursor<'a> {
         self.pos()
     }
 
+    /// Skip over space, tabs and newline characters, going backwards
+    /// Cursor position is set before the previous non-whitespace char
+    pub fn skip_whitespace_backwards(&mut self) -> usize {
+        while let Some(c) = self.get_prev_char() {
+            if !(c.is_whitespace()) {
```

Comment:
```suggestion
            if !c.is_whitespace() {
```
No need for extra parens.

Reply:
I'd also suggest transposing the if/else clauses. In Rust it's more common to see short circuiting/base case first.